bciers/apps/administration/app/components/operators/OperatorPage.tsx

```diff
@@ -90,9 +91,7 @@ export default async function OperatorPage({
   isCreating = false,
   operatorId,
 }: { isCreating?: boolean; operatorId?: UUID } = {}) {
-  const session = await auth();
-
-  const role = session?.user?.app_role;
+  const role = await getSessionRole();
```

Comment:
I was inspired to do this because if there's no role (it wasn't mocked in some of the vitests), then we should probably get an error, not set the isInternalUser boolean to either true/false

Reply:
Yes, one is for client components, one for server, I'll make a comment in the utils file